### PR TITLE
fix: flatten event type

### PR DESCRIPTION
**Pull request by jrandolf**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -53,6 +52,7 @@
         "terser": "5.19.2",
         "tslib": "2.6.1",
         "typescript": "5.1.6",
+        "urlpattern-polyfill": "9.0.0",
```
Comment (thiagowfx) [inline package-lock.json:55]: Is this change deliberate?

**Review by thiagowfx — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (jrandolf) [reply package-lock.json:55]: The `package-lock.json` wasn't updated before.
